superset-frontend/src/components/DvtMiniNavigation/dvt-mini-navigation.module.tsx

```diff
@@ -0,0 +1,82 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import { styled } from '@superset-ui/core';
+import { Link } from 'react-router-dom';
+
+// const rotate = keyframes`
+//   from {
+//     transform: rotate(0deg);
+//   }
+
+//   to {
+//     transform: rotate(360deg);
+//   }
+// `;
+
+interface FadeInIconProps {
+  $fadeIn: boolean;
+}
+
+const DvtMiniNavigationAnimatedIcon = styled.div<FadeInIconProps>`
+  transition: all 300ms;
+  transform: ${({ $fadeIn }) => ($fadeIn ? 'rotate(90deg)' : 'rotate(0)')};
+`;
+
+const StyledDvtMiniNavigation = styled.div`
+  height: 100px;
+  width: 100px;
```

Comment:
height ve width 100px; neden olsun ki?
eğer height 100px yaparsan eğer dizi link 10 adet item çıkarsa? height problem olur
eğer width 100px yaparsan yazısı "Dashboard 1" varya yerinde birisi "Reporting And With 1" bu yazısı çıkarsa 100px sığmaz mı? 

height ve width 100px ikise kaldırmalı. Display: inline-block kalsın

Reply:
completed